### PR TITLE
Add `Tilde` name to `KeyCode::Grave` comment

### PR DESCRIPTION
**Pull request by doonv**
# Objective

I had trouble finding the `` ` `` / `~` key, because I usually call it the tilde key.

## Solution

Add the name `Tilde` to `KeyCode::Grave`'s doc comment.

### CODE REVIEW

File: crates/bevy_input/src/keyboard.rs [hunk 1]
```diff
@@ -316,7 +316,7 @@ pub enum KeyCode {
     NumpadDivide,
     /// The `Equals` / `=` key.
     Equals,
-    /// The `Grave` / `Backtick` / `` ` `` key.
+    /// The `Grave` / `Backtick` / `Grave` / `` ` `` / `~` key.
```
Comment (alice-i-cecile) [inline crates/bevy_input/src/keyboard.rs:319]:
```suggestion
    /// The `Grave` / `Backtick` / `Tilde` / `` ` `` / `~` key.
```

Comment (doonv) [reply crates/bevy_input/src/keyboard.rs:319]: oh wait im stupid. sorry

**Review by alice-i-cecile — APPROVED** [review]

**Comment by mockersf** [comment]
this is not the same key on an azerty keyboard

**Comment by doonv** [comment]
> this is not the same key on an azerty keyboard

Yes, but most people use QWERTY. This is only a small comment to help people find the `` ` `` key.

**Comment by TimJentzsch** [comment]
Let's add a [doc alias](https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#alias) as well to make it easier to find in the search :)

**Review by TimJentzsch — APPROVED** [review]

**Comment by mockersf** [comment]
> Yes, but most people use QWERTY. This is only a small comment to help people find the `` ` `` key.

And it will confuse other people... Could you put the Tilde / ~. apart, and mention that it's only on qwerty keyboard then?

**Comment by devil-ira** [comment]
The update to winit 0.29 would make this PR obsolete. I'd take a look at the changes to keyboard input and documentation there

**Comment by Vrixyz** [comment]
> The update to winit 0.29 would make this PR obsolete. I'd take a look at the changes to keyboard input and documentation there

relevant code:
https://github.com/bevyengine/bevy/pull/8745/files#diff-249bdc151d76af3066180d266ee61c4e0138bb4cf67fe9e9e7602948d92c6b5eR133-R136

It doesn't add tilde, I'm not particularly against adding it, but from my understanding, keycodes tend to refer to the resulting qwerty key used without modifiers.

Also, that code where you want to add docs comes mostly exactly from winit, so I'd be more confortable with keeping an exact copy as much as possible, and upstream any differences within winit to keep differences to a minimum.

**Comment by doonv** [comment]
> The update to winit 0.29 would make this PR obsolete. I'd take a look at the changes to keyboard input and documentation there

So should I close this?

**Comment by alice-i-cecile** [comment]
Let's close this out: I don't think we're going to have a release where it ships.

**Pull request closed without merging**